src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexMatch.cs

```diff
@@ -63,8 +63,7 @@ public static Match Empty
         }
 
         internal Match(Regex regex, int capcount, String text, int begpos, int len, int startpos)
-
-        : base(text, new int[2], 0)
+            : base(text, new int[2], 0, "0")
```

Comment:
are we always going to have "0" name for the group with no specified name? what we used to return from GetGroupNames before this change?